packages/trpc/server/middlewares/sessionMiddleware.ts

```diff
@@ -103,23 +130,16 @@ export async function getUserFromSession(ctx: TRPCContextInner, session: Maybe<S
 
   const locale = user?.locale ?? ctx.locale;
 
-  const isOrgAdmin = !!user.profile?.organization?.members.filter(
```

Comment:
No longer needed, this (although 2000 iterations isn't "that" much - slows down every page depending on sessionMiddleware (tl;dr; speeds up all pages for logged in users)